.gitignore

```diff
@@ -19,3 +19,7 @@ yarn.lock
 
 # TypeScript
 jsconfig.json
+
+# JetBrains
```

Comment:
Found a better solution: `.git/info/exclude`: per-user gitignores. (Who knew? Not me till two minutes ago.)